extras/rya.pcj.fluo/pom.xml

```diff
@@ -74,6 +74,16 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <!--  Trivial listing of classes to be loaded via SPI -->
+                        <exclude>rya.pcj.functions.geo/src/main/resources/META-INF/services/org.openrdf.query.algebra.evaluation.function.Function</exclude>
```

Comment:
What about just `<exclude>**/resources/META-INF/services/**</exclude>` so that it isn't tied to just geo?

(note: I'm fine with it as is, too)